### PR TITLE
Dependency update.

### LINKED ISSUE #19: Add stricter constraint for L5 dependencies
See #17

### PR DESCRIPTION
**Pull request by kirkbushell**
Resolves #19

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,13 +22,13 @@
...
     "require": {
         "php": ">=5.4.0",
-        "beatswitch/lock": "0.1.*"
+        "beatswitch/lock": "0.1.*",
+        "illuminate/auth": "5.0.x",
+        "illuminate/database": "5.0.x",
+        "illuminate/support": "5.0.x"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.0",
```
Comment (driesvints) [inline composer.json]: Trailing comma here.

File: composer.json [hunk 2]
```diff
@@ -22,13 +22,13 @@
     },
     "require": {
         "php": ">=5.4.0",
-        "beatswitch/lock": "0.1.*"
+        "beatswitch/lock": "0.1.*",
+        "illuminate/auth": "5.0.x",
+        "illuminate/database": "5.0.x",
+        "illuminate/support": "5.0.x"
```
Comment (driesvints) [inline composer.json]: Can you use the `*` wildcard for these, please?

Comment (kirkbushell) [inline composer.json]: Which do you want me to use? .x is the preferred/documented approach on composer docs.

Comment (driesvints) [inline composer.json]: I use `*` for most of my projects so preferable that one.

I think at one point someone posted an unofficial poll on twitter which was more used and I think the `*` beat the `x` by a vast majority. But I have no idea anymore who posted it or where the results are.

Comment (kirkbushell) [inline composer.json]: Done.

**Comment by driesvints** [comment]
Thanks man!

**Pull request merged**